cg/meta/report/balsamic.py

```diff
@@ -49,7 +49,11 @@ def get_sample_metadata(
         """Fetches the sample metadata to include in the report"""
 
         sample_metrics = analysis_metadata.sample_metrics[sample.internal_id]
-        million_read_pairs = round(sample.reads / 2000000, 1) if sample.reads else None
+        million_read_pairs = (
+            round(sample.reads / 2000000, 1)
```

Comment:
```suggestion
            round(sample.reads / 2_000_000, 1)
```
grouping decimal numbers by thousands is more readable. What does this number stand for? Could this be a named constant to make it easier to understand why 2_000_000?

Reply:
I'm not sure if we can assign it to a constant since it is just a formatting value to represent the number of reads as million read pairs.

Didn't know about the thousands separators 👍